igvc_navigation/src/differential_drive/differential_drive.cpp

```diff
@@ -0,0 +1,44 @@
+#include <parameter_assertions/assertions.h>
+#include <ros/ros.h>
+
+#include "differential_drive.h"
+
+DifferentialDrive::DifferentialDrive()
+{
+  ros::NodeHandle nh;
+  ros::NodeHandle pNh("~");
+
+  assertions::param(pNh, "axle_length", axle_length_, 0.48);
+  assertions::param(pNh, "max_vel", max_vel_, 3.0);
+  ros::Subscriber mbf_twist = nh.subscribe("/cmd_vel", 1, &DifferentialDrive::twistToVelocity, this);
+  vel_pub_ = nh.advertise<igvc_msgs::velocity_pair>("/motors", 1);
+
+  ros::spin();
```

Comment:
Also, (this is something that should also be changed in a few other nodes), it's better to have the `ros::spin()` in the main than to have it in the constructor, since theoretically this would allow us to have multiple nodes running in an executable by constructing them in `main`.